src/assignment/components/modals/ManageAssignmentLabels.tsx

```diff
@@ -134,12 +136,29 @@ const ManageAssignmentLabels: FunctionComponent<ManageAssignmentLabelsProps> = (
 		setAssignmentLabels([...assignmentLabels.filter((labelObj) => labelObj.id !== id)]);
 	};
 
+	const labelsExceedMaxLenght = (labels: Assignment_Label_v2[]) => {
+		let isValid = true;
+
+		labels.forEach((label) => {
+			const exceedsMaxLength = (label.label?.length || 0) > MAX_LABEL_LENGTH;
+			if (exceedsMaxLength) {
+				isValid = false;
+			}
+		});
+
+		return isValid;
```

Comment:
find is more efficient than a foreach since it will stop after finding the first occurance
```suggestion
		return !labels.find(label => (label.label?.length || 0) > MAX_LABEL_LENGTH);
```